tests/test_smoke.py

```diff
@@ -1657,6 +1660,7 @@ def test_spot_cancellation_gcp():
 # ---------- Testing storage for managed spot ----------
 @pytest.mark.no_lambda_cloud  # Lambda Cloud does not support spot instances
 @pytest.mark.no_ibm  # IBM Cloud does not support spot instances
+@pytest.mark.no_oci
```

Comment:
Why do we disable this test?

Reply:
I noticed the referenced file managed_spot_with_storage.yaml in the test case is aws specific, also it uses aws s3 bucket.